geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/ContentFeatureSourceInfo.scala

```diff
@@ -0,0 +1,23 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.geotools
+
+import org.geotools.data.store.ContentFeatureSource
+import org.geotools.data.ResourceInfo
+
+
+trait ContentFeatureSourceInfo extends ContentFeatureSource {
```

Comment:
Definitely not for this ticket... 

... but, are there other shared goodies we could pull up into this trait?